test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -9440,6 +9442,45 @@ def forward(self, x):
         x = torch.randn(10, 5)
         self.run_test(M(), (x,))
 
+    def test_onnx_checker_invalid_graph(self):
+        op_source = """
+        #include <torch/script.h>
+
+        torch::Tensor custom_add(torch::Tensor self, torch::Tensor other) {
+          return self + other;
+        }
+
+        static auto registry =
+          torch::RegisterOperators("custom_namespace::custom_add", &custom_add);
+        """
+
+        torch.utils.cpp_extension.load_inline(
+            name="custom_add",
+            cpp_sources=op_source,
+            is_python_module=False,
+            verbose=True,
+        )
+
+        class CustomAddModule(torch.nn.Module):
+            def forward(self, a, b):
+                return torch.ops.custom_namespace.custom_add(a, b)
+
+        def invalid_symbolic(g, self, other):
+            return g.op("Add", self, other, invalid_attr_i=1)
+
+        register_custom_op_symbolic("custom_namespace::custom_add", invalid_symbolic, 9)
```

Comment:
Can we register a symbolic for a built-in aten op instead so we don't need the cpp_extension?
Not sure if it's possible to override an existing symbolic. If not then can we register a symbolic for an ATen op that is not supported in a particular op set version.

My concerns with the cpp extension are:
* complexity / hard to understand
* potentially slower to execute since it has to run a C++ compiler, etc

Reply:
Agree with your concerns. Using cpp_extension was the first idea of constructing an invalid operator. I've found out another proper way as you suggested so we can remove it now.